stable/artifactory-ha/README.md

```diff
@@ -1250,35 +1250,11 @@ Then install nginx-ingress with the values file you created:
 helm upgrade --install nginx-ingress --namespace nginx-ingress center/kubernetes-ingress-nginx/ingress-nginx -f values.yaml
 ```
 
-### Prometheus Metrics
+### Log Analytics
 
-If you want to enable Prometheus metrics you can use the `metrics` configuration options. By default this option requires that the Promtheus Operator already be deployed and the associated CRDs created.
+#### FluentD, Prometheus and Grafana
 
-The simplest way is to install Artifactory with the following command:
-
-```bash
-helm upgrade --install artifactory-ha --namespace artifactory-ha --set metrics.enabled=true center/jfrog/artifactory-ha
-```
-
-This will create a new service exposing the Prometheus metrics as well as a ServiceMonitor object for the Prometheus Operator to start scraping.
-
-The `ServiceMonitor` creation can be disabled by setting `metrics.serviceMonitor.enabled` to `false` for environments that are not using the Prometheus Operator.
-
-```bash
-helm upgrade --install artifactory-ha --namespace artifactory-ha --set metrics.enabled=true --set metrics.serviceMonitor.enabled=false center/jfrog/artifactory-ha
-```
-
-NOTE: Enabling this does NOT create a container which will actually parse the log files for metrics.  See the Fluentd section below.
-
-### Fluentd
-
-The suggested way to parse the log files for metrics is to install Fluentd as a sidecar container.  This can be done with the `fluentd` configuration options.
-
-```bash
-helm upgrade --install artifactory-ha --namespace artifactory-ha --set fluentd.enabled=true center/jfrog/artifactory-ha
-```
-
-This will install Artifactory with Fluentd running as a sidecar container sharing the persistent volume where the log files are written.
+To configure Prometheus and Grafana to gather metrics from Artifactory through the use of FluentD, please refer - https://github.com/jfrog/log-analytics-prometheus
```

Comment:
Can we change this to below:

To configure Prometheus and Grafana to gather metrics from Artifactory through the use of FluentD, please refer to the log analytics repo:

https://github.com/jfrog/log-analytics-prometheus

That repo contains a file `artifactory-ha-values.yaml` that can be used to deploy Prometheus, Service Monitor, and Grafana with this chart.